### ISSUE TITLE
Missing env attribute

### ISSUE THREAD

**Issue by mutairibassam** [AymanProjects/flutter-flavorz#11]
Code generation is only considering first object's attributes.

For example, the below json generated only `_name` and `versionNumber` without considering `test` for local env.
```json
{
    "environmentToRun": "prod",
    "environments": [
        {
            "_name": "dev",
            "versionNumber": "Dev 1.0.0"
        },
        {
            "_name": "local",
            "test": "test"
        }
    ]
}
```
Generated code:
```dart
class Environment {
  final String _name;
  final String versionNumber;
}
```

**Comment by AymanProjects** [comment]
Yes, the code generator will create the Environment class based on the the first element inside the json list. 

It is expected that all elements in the json list have an identical structure.

Thank you for the note. This will be mentioned in the readme file.

**Comment by mutairibassam** [comment]
Let's reconsider dynamic attributes in the future. Documentation should be precise and direct in this point, as sometimes people needs to add more attributes for QA and Production which is not exist for Mock or Development environments.

So, we should tell them an empty string would be enough for the current release.

**Comment by AymanProjects** [comment]
Fixed at #27
